src/OnlineRetailPortal.Contracts/Models/ProductList.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Contracts
+{
+
+        public class ProductList
```

Comment:
List word is confusing. This represents a data entity not list

Reply:
Why does this class exist if Product class is already there

Reply:
PageList is used for homepage user get a list of products it does not shows all product detail it only shows necessary details... in this pr, I am not using it anywhere.

Reply:
removed